client/src/containers/map/popup/index.tsx

```diff
@@ -16,21 +19,26 @@ const PopupContainer = () => {
       latitude={popup.lngLat.lat}
       longitude={popup.lngLat.lng}
       closeOnClick={false}
-      style={{
-        padding: 0,
-      }}
-      maxWidth="300px"
+      closeButton={false}
+      className="z-50 flex w-[360px] flex-col font-serif"
+      maxWidth="360px"
       onClose={() => setPopup(null)}
     >
-      <div className="pointer-events-none absolute left-0 top-0 h-4 w-full bg-gradient-to-b from-white" />
-      <div className="max-h-[49vh] space-y-2.5 overflow-y-auto overflow-x-hidden pr-8 text-slate-800 shadow-[0_20px_15px_rgba(0,0,0,0.1)]">
-        <div className="divide-y divide-slate-200">
+      <div className="p-6 pr-12">
+        <Button
```

Comment:
Can you review the design? I think it was changed since you implemented this: https://www.figma.com/file/DBXFTLd6nFBmg6fFE6P2cX/ORCaSa-Design?type=design&node-id=5744-20206&mode=design&t=3RFFcbTkLQPAJEhZ-4.